1001-2000/LIT1297Dersan.xml

```diff
@@ -77,13 +78,13 @@ It consists of a homily by John Chrysostom on Raphael, a collection of miracles
       <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Miracles">
       </div>
 
-      <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="HomilyRetua">
+      <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="HomilyRetua" corresp="LIT2124OntheF">
         <div type="textpart" subtype="incipit">
            <ab>ድርሳን፡ ዘ፬እንስሳ፡ ክቡራን፡ ዘደረሰ፡ ርቱዓ፡ ሃይማኖት፡</ab>
             </div>
       </div>
 
-      <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="HomilyChrysostom">
+      <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="HomilyChrysostom" corresp="LIT6404Dersan4recA">
         <div type="textpart" subtype="incipit">
            <ab>ድርሳን፡ ዘብፁዕ፡ ወቅዱስ፡ ዮሐንስ፡ አፈ፡ ወርቅ፡ ሊቀ፡ ጳጳሳት፡ ዘቍስጥንጥንያ፡
```

Comment:
Thank you, could you also encode this information as relation, either here or under the respective entries? 
That would be very helpful.